src/ScType.ts

```diff
@@ -1,33 +1,49 @@
 import {
   sc_type_const,
+  sc_type_var,
   sc_type_constancy_mask,
-  sc_type_dedge_common,
-  sc_type_edge_access,
-  sc_type_edge_fuz,
-  sc_type_edge_mask,
-  sc_type_edge_neg,
-  sc_type_edge_perm,
-  sc_type_edge_pos,
-  sc_type_edge_temp,
   sc_type_element_mask,
-  sc_type_link,
+  sc_type_connector,
+  sc_type_common_edge,
+  sc_type_arc,
+  sc_type_common_arc,
+  sc_type_membership_arc,
+  sc_type_pos_arc,
+  sc_type_neg_arc,
+  sc_type_fuz_arc,
+  sc_type_perm_arc,
+  sc_type_temp_arc,
+  sc_type_actual_arc,
+  sc_type_inactual_arc,
+  sc_type_node_link,
   sc_type_node,
-  sc_type_node_abstract,
   sc_type_node_class,
+  sc_type_node_superclass,
   sc_type_node_material,
-  sc_type_node_norole,
+  sc_type_node_no_role,
```

Comment:
if no_role is splitted with underscore, shouldn't sc-machine follow same style?

Reply:
fixed